react/week2/my-app/src/render.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+const list = [
+  {
+    id: 1,
+    description: "Get out of bed"
+  },
+  {
+    id: 2,
+    description: "Brush teeth"
+  },
+  {
+    id: 3,
+    description: "Eat breakfast"
+  },
+
+  {
+    id: 4,
+    description: "Go for a walk"
+  },
+
+  {
+    id: 5,
+    description: "Finish homework "
+  }
+];
+function RenderList(props) {
+  const isComplete = props.todos.completed;
+  return (
+    <li style={{ textDecoration: isComplete ? "line-through" : "none" }}>
+      {props.todos.description}
+
+      <input type="checkbox" onChange={props.handelCheckbox}></input>
+      <button onClick={props.deleteTodo}> Delete</button>
+    </li>
+  );
+}
+
+class TodoList extends React.Component {
+  state = {
+    listOfTodos: []
+  };
+
+  addTodo = this.addTodo.bind(this);
+  removeTodo = this.removeTodo.bind(this);
+  handelCheckbox = this.handelCheckbox.bind(this);
+
+  addTodo() {
+    const listIs = list.map(item => {
+      return item;
+    });
+
+    this.setState({
+      listOfTodos: [
+        ...this.state.listOfTodos,
+        list[Math.floor(Math.random() * list.length)]
+      ]
+    });
+  }
+
+  handelCheckbox(Index) {
```

Comment:
```suggestion
  handelCheckbox(index) {
```
Only components should be capitalised